### ISSUE TITLE
SUGGESTION - "Charging" Warp

### ISSUE THREAD

**Issue by CaptLapadite** [daid/EmptyEpsilon#399]
Jump and Warp are quite different and interesting ways for long distance travel, each with it's advantage and drawbacks. In pvp, both are appealing and offer diversity on the battlefield. 

Our tests in pvp have shown that warp can also be used in an odd way that seems to get out of the boundary of it's own purpose (long distance travel). It's instead used as a short burst, exactly the same as the "maneuver boost" pad, but better and with unlimited use. It put a lot of inequality in a fight.

The suggestion is to add a small charging time when using warp, during which time the ship could not move (or have diminished speed/maneuver). In fact it could simply be the same as when you Jump, but for 3 seconds. This 3 secs, would go faster if the warp engine is overcharged by engineering. That would aims to make the player use the Warp engine in a more strategic way, and to leave other players a chance to react during combat.

Here's an exemple of warp drive in Rebel galaxy :

https://www.youtube.com/watch?v=K0ZMAlcD7XA

**Comment by daid** [comment]
I agree that the warp drive needs some work. I always use ships with a jump drive for my players. While AI ships do not abuse the warp drive.

This delay before activating could indeed fix the main issue I have with the warpdrive, which is it's huge combat advantage.

**Comment by Wrongtown** [comment]
👍
